src/ui/application-window.vala

```diff
@@ -395,10 +451,26 @@ namespace Peek.Ui {
 
     private void update_input_shape () {
       // Set an input shape so that the recording view is not clickable
+
+
       var window_region = create_region_from_widget (recording_view.get_toplevel ());
       var recording_view_region = create_region_from_widget (recording_view);
       window_region.subtract (recording_view_region);
 
+      //popover menu
+      if (pop_format.visible) {
+	    var theme = new GLib.Settings ("org.gnome.desktop.interface");//grab the users' settings
+	    var theme_name = theme.get_string ("gtk-theme");//find the users' theme
```

Comment:
I'd add the above two lines to a static helper function `get_theme_name()` in desktop-integration.vala. This way it can be reused in application.vala.